kpatch-build/kpatch-build

```diff
@@ -430,15 +454,16 @@ fi
 
 [[ -z $TARGETS ]] && TARGETS="vmlinux modules"
 
-PATCHNAME=$(basename "$PATCHFILE")
-if [[ "$PATCHNAME" =~ \.patch$ ]] || [[ "$PATCHNAME" =~ \.diff$ ]]; then
-	PATCHNAME="${PATCHNAME%.*}"
-fi
+# If no kpatch module name was provided on the command line, derive one
+# from the first .patch filename
+if [[ -z $MODNAME ]] ; then
+	MODNAME=$(basename "${PATCH_LIST[0]}")
```

Comment:
Naming the patch based on the first .patch name could be confusing.  Maybe just call it kpatch-patch?

Reply:
That seems like a reasonable default.